dashboard/app/models/levels/blockly.rb

```diff
@@ -609,4 +610,18 @@ def localized_shared_blocks(level_objects)
     end
     level_objects_copy
   end
+
+  def update_goal_override
+    if goal_override.present? && goal_override.is_a?(String)
```

Comment:
super nit: we can check for presence and type at the same time--
```suggestion
    if goal_override&.is_a?(String)
```